_includes/navbar.html

```diff
@@ -3,11 +3,14 @@
 
 <nav id="header">
   <span id="left-side">
-    <a href="{{site.baseurl}}/" class="navbar_tab">Home</a>
-    <a href="{{site.baseurl}}/main/" class="navbar_tab">ACLE</a>
-    <a href="{{site.baseurl}}/mve_intrinsics/" class="navbar_tab">MVE</a>
-    <a href="{{site.baseurl}}/morello/" class="navbar_tab">Morello</a>
-    <a href="{{site.baseurl}}/neon_intrinsics/" class="navbar_tab">Neon</a>
+    {% for nav in site.data.navigation %}
+      <a href="{{site.baseurl}}{{nav.url}}"
+        {% if page.url == "/" and nav.url == "/"%} class="navbar_active_tab"
+        {% elsif nav.url != "/" and page.url contains nav.url %} class="navbar_active_tab"
```

Comment:
Could we simplify it to this, also adding a comment that say we need special treatment of the home `nav.url == "\"` because `/` is in all urls?
```suggestion
        {% if page.url == nav.url %} class="navbar_active_tab"
        <!-- We need special treatment of the home nav.url == "\" because `/` is in all URLs. -->
        {% elsif  page.url contains nav.url and nav.url != "/" %} class="navbar_active_tab"
```

Reply:
Sure